pkg/pvc/pvc_protected_entity.go

```diff
@@ -283,6 +286,67 @@ func (this PVCProtectedEntity) getProtectedEntityForPV(ctx context.Context, pv *
 				return nil, errors.Errorf("Unexpected access mode, %v, for Persistent Volume %s", pv.Spec.AccessModes[0], pv.Name)
 			}
 		}
+	} else if pv.Spec.VsphereVolume != nil {
+		if pv.Spec.AccessModes[0] != core_v1.ReadWriteOnce {
+			return nil, errors.Errorf("Unexpected access mode, %v, for Persistent Volume %s", pv.Spec.AccessModes[0], pv.Name)
+		}
+		if pv.Annotations["pv.kubernetes.io/migrated-to"] == VSphereCSIProvisioner {
+			volumePath := pv.Spec.VsphereVolume.VolumePath
+
+			// Migrated volume must be of type ivd
```

Comment:
The comment says must be of type ivd, will you be adding checks for it?

Reply:
throw error if not ivd pe type.